.github/workflows/CI.yml

```diff
@@ -22,9 +22,13 @@ jobs:
        run: |
         python -m pip install --upgrade pip
         pip install flake8
+        pip install isort
      - name: Analysing the code with flake8
        run: |
         flake8 $(git ls-files '*.py')
+     - name: Checking import sorting
+       run: |
+        isort . --check --diff
```

Comment:
Nit: Is `--diff` needed with check?

Reply:
No, it's not required, but it means that we get helpful messages instead of just "didn't match"